metadata.json

```diff
@@ -124,7 +124,8 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "20.04"
+        "20.04",
+				"22.04"
```

Comment:
Please also add it here: https://github.com/voxpupuli/puppet-zabbix/blob/master/spec/spec_helper_methods.rb#L16